builder.py

```diff
@@ -306,26 +307,27 @@ def sort_version_by_release_date(i):
 
             link = "{}/thunderbird/{}/releasenotes/".format(settings.CANONICAL_URL, version)
 
+
             # Mix in our notes for the template
             self._env.globals.update(**note)
             content = content_template.render({'version_number': version, 'link': link})
 
             entry = feed.add_entry()
+            entry.id(link)
             entry.title(title)
             entry.link({'href': link})
-            entry.description(description=content)
-            entry.content(content, type="CDATA")
+            entry.content(content, type='html')
```

Comment:
Maybe type="xhtml" here? Then it won't have to do all of the funny escaping. The template will need to be valid xhtml probably.

Either way, there's a lot of empty space after rendering the template. Possibly run the content through something like https://pypi.org/project/htmlmin/, or use something like https://pypi.org/project/pyquery/ to build programatically? Neither option is great since they have dependencies...

Reply:
I went with htmlmin. It'd be nice to eventually minify everything, since it seems to be a common issue through the site. There are settings for jinja to strip white spaces, but I couldn't seem to get it working with just the feed. (or at all, actually :thinking:)

As for the xhtml, the python markdown library seems to be acting weird with missing quotes around the src attribute. I'm poking at it a bit more though. Although I don't think the funny escaping is a particular issue, unless you've encountered problems with it in a reader I haven't tested.

Reply:
The escaping is a readability problem more than anything else. But if we are minifying too that argument doesn't hold up. :-)

Reply:
The Jinja version on production is possibly too old for whitespace stripping?

Reply:
Locally (from requirements-dev) I've got 2.11.3 installed, and looks like it should be supported: https://jinja.palletsprojects.com/en/2.11.x/templates/#whitespace-control (lstrip_blocks, and trim_blocks). But alas, settings those in the Environment constructor does nothing to the feeds output. 

There's a few other issues I've encountered trying to set it to xhtml, mostly crypticly set error messages (despite the modified builds producing xhtml valid markup :nerd_face:).